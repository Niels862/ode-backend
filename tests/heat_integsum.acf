let alpha = 0.4;
let beta = 2;

chip {
    io: [
        input,
        input,
        output,
    ],
    cabs: [
        cab 1 with clocks 1, - {
            cams: [
                Integrator as integ1 {
                    inputs: 3,
                    integ_const1: alpha,
                    integ_const2: alpha * beta,
                    integ_const3: alpha,
                    invert2: true,
                },
                Integrator as integ2 {
                    inputs: 3,
                    integ_const1: alpha,
                    integ_const2: alpha * beta,
                    integ_const3: alpha,
                    invert2: true,
                },
            ]
        },
        cab 2 with clocks 1, - {
            cams: [
                SampleAndHold as sah1 {},
                SampleAndHold as sah2 {},
            ]
        }
    ],
    routing: [
        io1 -> integ1:1,
        sah1 -> integ1:2,
        sah2 -> integ1:3,
        integ1 -> sah1,

        sah1 -> integ2:1,
        sah2 -> integ2:2,
        io2 -> integ2:3,
        integ2 -> sah2,

        sah2 -> io3,
    ],
}
