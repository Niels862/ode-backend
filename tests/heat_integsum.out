0 0 0 0 32 4 0 2 5 0 0 64 0 0 81 255 15 241 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 64 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 255 0 80 0 0 0 0 0 18 0 0 0 0 0 16 0 0 64 0 0 64 0 0 0 0 0 0 0 0 0 0 0 0 250 25 50 25 250 25 50 25 0 0 0 0 0 0 0 0 0 32 0 32 1 30 1 130 0 48 0 16 1 31 1 129 12 0 0 0 0 0 0 0 0 0 0 0 1 129 1 130 1 241 1 130 0 5 1 225 1 129 0 5 1 145 1 129 0 0 0 0 255 255 255 255 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 32 1 130 0 48 1 129 12 0 2 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 5 1 42 1 40 0 5 1 59 1 24 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0